vllm/v1/engine/async_llm.py

```diff
@@ -56,11 +59,16 @@ def __init__(
 
         self.log_requests = log_requests
         self.log_stats = log_stats
-        self.stat_loggers: list[StatLoggerBase] = []
+        self.stat_loggers: dict[str, StatLoggerBase] = dict()
         if self.log_stats:
-            if logger.isEnabledFor(logging.INFO):
-                self.stat_loggers.append(LoggingStatLogger())
-            self.stat_loggers.append(PrometheusStatLogger(vllm_config))
+            if stat_loggers is not None:
+                self.stat_loggers = stat_loggers
+            else:
+                if logger.isEnabledFor(logging.INFO):
+                    self.stat_loggers[STANDARD_LOGGING_LOGGER_NAME] = (
+                        LoggingStatLogger())
+                self.stat_loggers[PROMETHEUS_LOGGING_LOGGER_NAME] = (
+                    PrometheusStatLogger(vllm_config))
```

Comment:
How about we move all of this into the loggers module so that we just do e.g.

```
    self.stat_loggers = loggers.setup(vllm_config, stat_loggers)
```